admin_terraform_version_integration_test.go

```diff
@@ -97,6 +97,7 @@ func TestAdminTerraformVersions_List(t *testing.T) {
 
 func TestAdminTerraformVersions_CreateDelete(t *testing.T) {
 	skipUnlessEnterprise(t)
+	skipUnlessBeta(t)
```

Comment:
Is it just the first t.Run test with arches that needs to be skipped unless beta?